board-report-2022-04.md

```diff
@@ -0,0 +1,49 @@
+## Description:
+
+The mission of Apache Arrow is the creation and maintenance of
+software related to columnar in-memory processing and data interchange
+
+## Issues:
+
+There are no issues requiring board attention.
+
+## Membership Data:
+
+Apache Arrow was founded 2016-01-20 (6 years ago)
+There are currently 72 committers and 39 PMC members in this project.
+The Committer-to-PMC ratio is roughly 3:2.
+
+Community changes, past quarter:
+- QP Hou was added to the PMC on 2022-02-17
+- Yibo Cai was added to the PMC on 2022-01-04
+- Alessandro Molina was added as committer on 2022-01-05
+- Kun Liu was added as committer on 2022-03-09
+- Jacob Quinn was added as committer on 2022-02-23
+- Raphael Taylor-Davies was added as committer on 2022-03-24
+- Xudong Wang was added as committer on 2022-03-16
+- Yijie Shen was added as committer on 2022-03-09
+
+## Project Activity:
+
+Recent releases:
+
+- RS-11.0.0 was released on 2022-03-21.
+- JULIA-2.2.1 was released on 2022-03-08.
+- RS-10.0.0 was released on 2022-03-08.
+- RS-9.1.0 was released on 2022-02-22.
+- RS-DF-7.0.0 was released on 2022-02-17.
+- RS-9.0.2 was released on 2022-02-13.
+- 7.0.0 was released on 2022-02-03.
+- RS-8.0.0 was released on 2022-01-25.
+- RS-7.0.0 was released on 2022-01-12.
+
+## Community Health:
+
+- We've released a new version of Apache Arrow Julia after it was re-donated.
+- We'll start IP clearance for Apache Arrow Flight JDBC driver donation.
```

Comment:
```suggestion
- We'll start IP clearance for Apache Arrow Flight SQL JDBC driver donation.
```

Just a nit. Thanks for putting this together!

Reply:
Good catch!